### ISSUE TITLE
Need a way to see what Machine size presets are available (and what the limits are for RAM)

### ISSUE THREAD

**Issue by catflydotio** [superfly/flyctl#1646]
There's a clue in flyctl that there are preset sizes, but no list of what they are, as far as I remember seeing.

Nomad only:
```
$ fly platform vm-sizes
Update available 0.0.451-pre-5 -> v0.0.451.
Run "fly version update" to upgrade.
NAME                    CPU CORES       MEMORY 
shared-cpu-1x           1               256 MB
dedicated-cpu-1x        1               2 GB  
dedicated-cpu-2x        2               4 GB  
dedicated-cpu-4x        4               8 GB  
dedicated-cpu-8x        8               16 GB 
```

**Comment by tvdfly** [comment]
related https://github.com/superfly/flyctl/issues/1547

**Comment by billyb2** [comment]
I think this was fixed in 54a6dcd2c
